### ISSUE TITLE
[LSP] should shutdown return new Object instead of null

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-core#240]
LSP Shutdown gives

```
Received message which is neither a response nor a notification message:
{
    "id": "3",
    "jsonrpc": "2.0"
}
```

**Comment by cdietrich** [comment]
see https://github.com/eclipse/lsp4j/issues/18

**Comment by cdietrich** [comment]
workaround:

```
	public static class LanguageServerImpl2 extends LanguageServerImpl {
		  @Override
		  public CompletableFuture<Object> shutdown() {
		    return CompletableFuture.<Object>completedFuture(new Object());
		  }
	}
```

**Comment by cdietrich** [comment]
@spoenemann @svenefftinge what do you think?

**Comment by cdietrich** [comment]
Fixed with Xtext 2.11
